library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -324,7 +327,7 @@ class Client() {
                     isSecure = false,
                     db = dbPath,
                     encryptionKey = retrievedKey.encoded,
-                    accountAddress = account.address.lowercase(),
+                    accountAddress = accountAddress.lowercase(),
```

Comment:
I should remove the lowercase here as well.

Reply:
No changes needed in libxmtp for this one right?

Reply:
@richardhuaaa just removed this lowercase and now the tests fail so I think something on the rust side could be helpful. But not a huge deal we can leave the lowercaseing here

Reply:
What was the error message?

Reply:
Never mind, found it